docs/NSR.md

```diff
@@ -457,37 +457,77 @@ The CA SHOULD ensure incident response plans minimally include:
+* malicious software;
+* phishing and social engineering;
+* denial of service;
+* unauthorized access; and
+* malicious data injection.
 
-      i. vulnerabilities with high CVSS scores, starting with the vulnerabilities the CA determines are the most critical (such as those with a CVSS score of 10.0) and
-      ii. systems that lack sufficient compensating controls that, if the vulnerability were left unmitigated, would allow external system control, code execution, privilege escalation, or system compromise; or
+#### 4.3 Vulnerability Correction Process
 
-   3. Document the factual basis for the CA’s determination that the vulnerability does not require remediation because
+The CA MUST document and follow a vulnerability correction process that includes:
 
-      i. the CA disagrees with the NVD rating,
-      ii. the identification is a false positive,
-      iii. the exploit of the vulnerability is prevented by compensating controls or an absence of threats; or
-      iv. other similar reasons.
+   1. identification;
+   1. review;
+   1. response; and
+   1. remediation.
 
-g. Apply recommended security patches to Certificate Systems within six (6) months of the security patch's availability, unless the CA documents that the security patch would introduce additional vulnerabilities or instabilities that outweigh the benefits of applying the security patch.
+A vulnerability is remediated when the CA has:
+
+* fixed the vulnerability such that the vulnerability is no longer present; or
+* confirmed the impact of the vulnerability and documented why the vulnerability does not impact the CA's security posture.
+
+#### 4.4 Vulnerability Identification
+
+The CA's vulnerability identification process MUST include monitoring for relevant security advisories and penetration testing.
+
+##### 4.4.1 Penetration Testing
+
+As part of the identification component of the CA's vulnerability correction process, the CA MUST define and follow a program for performing penetration tests that ensures:
+
+   1. penetration tests are performed:
+      * at least on an annual basis; and
+      * after infrastructure or application changes that are organizationally defined as significant; and
+   2. penetration tests are performed by a person or entity (or collective group thereof) with the requisite skills, tools, proficiency, code of ethics, and independence; and
+   3. vulnerabilities identified during the penetration test are remediated using the vulnerability correction process in [Section 4.3](#43-vulnerability-correction-process).
+
+#### 4.5 Vulnerability Management Timeframe
+
+The CA MUST establish one or more timeframes for reviewing, responding to, and remediating all identified vulnerabilities.
+
+Each timeframe MUST be established based on a risk assessment performed by the CA.
+
+The risk assessment MUST be based on a documented security analysis.
```

Comment:
A security analysis as listed here seems like the precursor or simply a step in creating the risk assessment. While we give some examples below of what goes into that, it's still not really clear what the 'documented security analysis' itself really is. Are we expecting documentation of a meeting? A work ticket in which there are comments about performing the steps listed below? Or just a risk assessment that was built from this work is the documentation that this work was done? What is the thing that constitutes the documented security analysis?

Reply:
Fundamentally I think this is the documentation providing evidence which justifies the conclusions of the risk assessment. That is, the documented security analysis would be something CAs should be able to point to to understand (and explain) why the timeframes here have been established.

The specifics of what this includes, how it's formatted, where it's kept, whose responsibility it is, whether or when it's updated, etc. are not included in the scope of this ballot. I would be supportive of adding language which expounds on this process, if that would be useful for CAs or the broader industry, but I don't currently believe such language is necessary for the requirements here to be correctly followed and provide value to the ecosystem.